_components/tag/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/tag/accessibility-tests/
+layout: accessibility-test
+component:
+  name: tag
+title: Tag accessibility tests
+category: Components
+lead: Any non-interactive USWDS tag should pass these manual accessibility tests.
```

Comment:
Should we add `component` here? We discussed adding this in the [Prose checklist PR](https://github.com/uswds/uswds-site/pull/2613#discussion_r1570974447)

```suggestion
lead: Any non-interactive USWDS tag component should pass these manual accessibility tests.
```

Reply:
Good catch. Updated in [be3b025](https://github.com/uswds/uswds-site/pull/2645/commits/be3b02538509ccdb30364439243a69cda74450c0).